src/ArtistList.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import ArtistDetail from './ArtistDetail';
+import Artist from './Artist';
+
+class ArtistList extends React.Component{
+    state ={
+        sortType: '',
+        artists: []
+    }
+
+    componentDidMount() {
+        this.setState({
+            artists: ArtistDetail
+        })
+    }
+
+
+    //Vote handler
+    voteHandler = (artistId) => {
+        const voteArtist = this.state.artists.map(artist => {
+            if(artistId === artist.id) {
+                return Object.assign({}, artist, {
+                    votes: artist.votes + 1
+                })
+            }
+            else {
+                return artist
+            }
+        })
+        this.setState({
+            artists: voteArtist
+        })
+    }
+
+    voteHandlerDown = (artistId) => {
+        const voteArtist = this.state.artists.map(artist => {
+            if(artistId === artist.id) {
+                return Object.assign({}, artist, {
+                    votes: artist.votes - 1
+                })
+            }
+            else {
+                return artist
+            }
+        })
+        this.setState({
+            artists: voteArtist
+        })
+    }
+
+render(){
+
+    //sort artists with highest votes
+    const {artistname, sortType} = this.state;
+
+    const sortedArtist = this.state.artists.sort((a,b) => {
+        // if(b.votes === a.votes){
+        //     const namesSorted = this.state.artists.sort((a,b) => {
+        //     const isReversed = (sortType === 'asc') ? 1 : -1;
+        //     return isReversed * a.name.localeCompare(b.name);
+        //     }
+        // },
+        return b.votes - a.votes;
+        
+    })
```

Comment:
When using arrow functions, you can avoid return statement if it's a one line code and do it like this:
```suggestion
    const sortedArtist = this.state.artists.sort((a,b) => b.votes - a.votes)
```
Note that you shouldn't define a block using `{}` when doing this.

Reference: https://jaketrent.com/post/javascript-arrow-function-return-rules/